packages/css/src/types.ts

```diff
@@ -511,7 +511,7 @@ export type ColorMode = {
 }
 
 export interface Theme {
-  breakpoints?: ObjectOrArray<number | string | symbol>
+  breakpoints?: Array<string>
```

Comment:
According to the documentation this is only ever a `string[]`. Also was necessary to fix type issues in match-media

Reply:
I'd guess the types for this might have originated in older versions of Styled System FWIW -- might be some other holdovers from that theme interface to keep an eye out for